docs/cloud-native-security/cspm-get-started-aws.asciidoc

```diff
@@ -28,11 +28,31 @@ This page explains how to get started monitoring the security posture of your cl
 [[cspm-setup]]
 == Set up CSPM for AWS
 
-You can set up CSPM for AWS either by enrolling a single cloud account, or by enrolling an organization containing multiple accounts. Either way, first you will add the CSPM integration, then enable cloud account access.
+You can set up CSPM for AWS either by enrolling a single cloud account, or by enrolling an organization containing multiple accounts. Either way, first you will add the CSPM integration, then enable cloud account access. Two deployment technologies are available: agentless, and agent-based. <<cspm-aws-agentless, Agentless deployment>> allows you to collect cloud posture data without having to manage the deployment of an agent in your cloud. <<cspm-aws-agent-based, Agent-based deployment>> requires you to deploy and manage an agent in the cloud account you want to monitor. 
+
+[discrete]
+[[cspm-aws-agentless]]
+== Agentless deployment 
+beta::[]
+
+. From the Elastic Security *Get started* page, click *Add integrations*.
+. Search for `CSPM`, then click on the result.
+. Click *Add Cloud Security Posture Management (CSPM)*.
+. Select *AWS*, then either *AWS Organization* to onboard multiple accounts, or *Single Account* to onboard an individual account.
+. Give your integration a name that matches the purpose or team of the AWS account/organization you want to monitor, for example, `dev-aws-account`.
+. Click **Advanced options** then select **Agentless (BETA)**.
+. Next, you'll need to authenticate to AWS. Two methods are available:
+.. Option 1: Direct access keys / CloudFormation (Recommended). Under **Preferred method** select **Direct access keys**. Expand the **Steps to Generate AWS Account Credentials** section, then follow the instructions that appear to automatically create the necessary credentials using CloudFormation.
```

Comment:
If the extra spaces aren't part of the AWS UI, they're prob not needed. I also recommended a minor ordering change in the second sentence because the original could be interpreted as "the instructions give the impression of credentials being automatically created." This also applies across the other updated areas if you choose to accept it. 

```suggestion
.. Option 1: Direct access keys/CloudFormation (Recommended). Under **Preferred method**, select **Direct access keys**. Expand the **Steps to Generate AWS Account Credentials** section, then follow the displayed instructions to automatically create the necessary credentials using CloudFormation.
```